composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "messagebird/php-rest-api",
     "description": "MessageBird REST API client for PHP",
-    "version": "1.4.1",
+    "version": "1.5.0",
```

Comment:
Yeah, we can remove this. As mentioned in the Composer documentation for the `version` tag: 

> Optional if the package repository can infer the version from somewhere, such as the VCS tag name in the VCS repository. In that case it is also recommended to omit it.

https://getcomposer.org/doc/04-schema.md#version